routes/v1.js

```diff
@@ -3,11 +3,25 @@ var debug       = require('debug')('hooks:routes:v1');
 var Promise     = require('promise');
 var slugid      = require('slugid');
 var taskcluster = require('taskcluster-client');
-var utils       = require('../hooks/utils');
+var nextDate    = require('../hooks/nextdate');
 
 var api = new base.API({
   title:         "Hooks API Documentation",
-  description:   "Todo",
+  description:   [
+    "Hooks are a mechanism for creating tasks in response to events.",
+    "",
+    "Hooks are identified with a hook-group ID and a hook ID.",
```

Comment:
nit, other places we typically write in camelCase for identifiers: `hookGroupId` and `hookId`...